text/chapter6.md

```diff
@@ -123,16 +125,14 @@ No type class instance was found for
   Data.Show.Show (Int -> Int)
 ```
 
+Type class instances can be defined in one of two places: in the same module that the type class is defined, or in the same module that the type "belonging to" the type class is defined. An instance defined in any other spot is called an "orphan instance" and is not allowed by the PureScript compiler. Some of the exercises in this chapter will require you to copy the definition of a type into your MySolutions module so that you can define type class instances for that type.
```

Comment:
Again, feel free to nix my extra commentary.  I thought it might confuse some readers why in every chapter up to this point, they've been given all the code they need already and can just start coding up the exercises, but in this chapter they suddenly have to copy+paste a bunch of types into their code.